ts/config.ts

```diff
@@ -43,8 +43,7 @@ export const isDebugBiometricIdentificationEnabled =
   Config.DEBUG_BIOMETRIC_IDENTIFICATION === "YES";
 export const instabugToken: string = Config.INSTABUG_TOKEN;
 
-export const bonusVacanzeEnabled: boolean =
-  Config.BONUS_VACANZE_ENABLED === "YES";
+export const bonusVacanzeEnabled: boolean = true;
```

Comment:
I guess this is a leftover from development?

Reply:
Yes, thank you. Restored in [2ce0210](https://github.com/pagopa/io-app/pull/3267/commits/2ce021047efde10750d3da5a137a4ce29215c1a5)